### PR TITLE
PT-1205 Display data based on timezone objects

### PR DESCRIPTION
**Pull request by yvon-dblg**
With https://github.com/mdblp/tideline/pull/36

### CODE REVIEW

**Review by franck-fourel — COMMENTED** [review]
I have not yet looked at the code but testing locally I got some strange behavior, such as the background hours are UTC while displayed hours in tooltips look correct
image

**Review by BigFab — APPROVED** [review]
Some general comments, which are not really viz or tideline but both. Not sure it is feasible and whether it makes sense or not. But :)
- as a user, I think I would prefer seeing my data with the information "summer time/winter time" instead of "UTC+x" or "UTC+x+1"
- for the time change at the top of the timeline, I would also prefer seeing something like "winter time change" rather then "time change"

I was able to test thoroughly. I think it is working pretty well. I just have some doubts concerning the fact that having a row of data (e.g. "cbg") set to "UTC" timzeone will make the "timechange" icons disappear. This might be something to discuss further.
Waiting for that I am giving my approval. Not sure anyone else would like to test this before we merge? @franck-fourel ?

**Comment by yvon-dblg** [comment]
> 
> 
> Some general comments, which are not really viz or tideline but both. Not sure it is feasible and whether it makes sense or not. But :)
> 
>     * as a user, I think I would prefer seeing my data with the information "summer time/winter time" instead of "UTC+x" or "UTC+x+1"
> 
>     * for the time change at the top of the timeline, I would also prefer seeing something like "winter time change" rather then "time change"

I will try to see if it is possible.
If I recall correctly, we have the information like CET/CEST for Central Europe, but elsewhere it is a different name. So I'm not sure I want to do a match for all timezone...

**Pull request merged**
